### PR TITLE
Added Huggingface 🤗 model and spaces links

### PR DESCRIPTION
**Pull request by buoi**
Added my contribution for the Huggingface Keras Sprint!

Model and space links at the end of example: Structured Data Classification from Scratch

### CODE REVIEW

**Comment by buoi** [comment]
I added a table with HuggingFace links below the Keras code example, very similar to here https://github.com/keras-team/keras-io/pull/912

I did some pull-request closing-reopening mess to add my email for the Google Contributor License Agreement.

File: examples/structured_data/md/structured_data_classification_from_scratch.md [hunk 1]
```diff
@@ -472,13 +472,12 @@ Let's visualize our connectivity graph:
 keras.utils.plot_model(model, show_shapes=True, rankdir="LR")
 ```
 
+<div class="k-default-codeblock">
+```
```
Comment (fchollet) [inline examples/structured_data/md/structured_data_classification_from_scratch.md:476]: This block change is incorrect, please revert

**Review by fchollet — COMMENTED** [review]
Thanks for the PR!

**Comment by pcoet** [comment]
@buoi Thanks for the PR. Are you still working on this? Please let us know. Otherwise we'll close the request.

**Pull request closed without merging**